src/main/java/it/pagopa/pn/paperchannel/middleware/queue/consumer/handler/CustomAggregatorMessageHandler.java

```diff
@@ -0,0 +1,99 @@
+package it.pagopa.pn.paperchannel.middleware.queue.consumer.handler;
+
+
+import it.pagopa.pn.paperchannel.generated.openapi.msclient.pnextchannel.v1.dto.DiscoveredAddressDto;
+import it.pagopa.pn.paperchannel.generated.openapi.msclient.pnextchannel.v1.dto.PaperProgressStatusEventDto;
+import it.pagopa.pn.paperchannel.generated.openapi.server.v1.dto.StatusCodeEnum;
+import it.pagopa.pn.paperchannel.mapper.RequestDeliveryMapper;
+import it.pagopa.pn.paperchannel.mapper.common.BaseMapperImpl;
+import it.pagopa.pn.paperchannel.middleware.db.dao.EventMetaDAO;
+import it.pagopa.pn.paperchannel.middleware.db.entities.PnDeliveryRequest;
+import it.pagopa.pn.paperchannel.middleware.db.entities.PnDiscoveredAddress;
+import it.pagopa.pn.paperchannel.middleware.db.entities.PnEventMeta;
+import it.pagopa.pn.paperchannel.middleware.queue.consumer.MetaDematCleaner;
+import it.pagopa.pn.paperchannel.service.SqsSender;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import reactor.core.publisher.Mono;
+
+import static it.pagopa.pn.paperchannel.utils.MetaDematUtils.buildMetaRequestId;
+import static it.pagopa.pn.paperchannel.utils.MetaDematUtils.buildMetaStatusCode;
+
+@Slf4j
+public class CustomAggregatorMessageHandler extends SendToDeliveryPushHandler {
```

Comment:
è possibile estendere AggregatorMessageHandler in modo da recuperare il codice comune (esempio enrichEvent e (se possibile) una parte del contenuto di handlemessage)

Reply:
abbiamo fixato la classe estendendo AggregatorMessageHandler